scrapers/AdultDvdMarketPlace.yml

```diff
@@ -0,0 +1,23 @@
+name: AdultDvdMarketPlace
+
+movieByURL:
+  - action: scrapeXPath
+    url:
+      - adultdvdmarketplace.com/xcart/adult_dvd/dvd_view.php?adult_dvd_id=
+    scraper: movieScraper
+
+xPathScrapers:
+  movieScraper:
+    movie:
+      Name: //meta[@property="og:title"]/@content
+      Director: //span[contains(text(),"Director")]/following-sibling::a/text()
+      Date:
+        selector: //span[contains(text(),"Released")]/following-sibling::text()
+        postProcess:
+          - parseDate: 01/2006
+      Synopsis: //h3[contains(text(), "Description")]/following-sibling::p/text()
```

Comment:
`/text()` should be removed as it ommits bold or other formatted parts of the paragraph
eg `https://www.adultdvdmarketplace.com/dvd_view_362671.html`
```yaml
      Synopsis: //h3[contains(text(), "Description")]/following-sibling::p
```
is enough